rendering_styles/default.render.xml

```diff
@@ -9183,7 +9200,7 @@
 				</case>
 				<case minzoom="12" tag="leisure" value="garden" color="$gardenColor"/>
 				<apply>
-					<switch minzoom="16" showAccess="true">
+					<switch minzoom="16" showAccess="true" rClass=".road.access">
 						<case additional="access=private" color_2="$accessPrivateColor"/>
```

Comment:
should it be clasess by access tag? .road.access.private or dynamic class by acces stag